pkg/database-types/constants.go

```diff
@@ -34,7 +33,9 @@ const Region = "region"
 const S3 = "s3"
 const Scheme = "scheme"
 const SecretAccessID = "secret_access_id"
+const SecretKey = "secret_key"
```

Comment:
why do we need 2 cases of "Mysql" ?

Reply:
`Mysql` is the name used by OpenMetadata
`mysql` is the connection name in the `fybrik` taxonomy